src/data/quot.lean

```diff
@@ -64,6 +64,14 @@ lemma factor_mk_eq {α : Type*} (r s : α → α → Prop) (h : ∀ x y, r x y 
 
 variables {γ : Sort*} {r : α → α → Prop} {s : β → β → Prop}
 
+/-- **Alias** of `quot.lift_beta`. -/
```

Comment:
```suggestion
```

That's not a particularly helpful docstring (= it doesn't help me at all on hover).  Either remove or turn into a regular comment with more details (= we want to use a different convention than core).  A mod-doc comment would also be ok if you want the explanation be in the api docs.

Reply:
I added this docstring because that's what the `alias` command would have done, if we were actually able to use it here.

Reply:
Ok, then I won't complain about it here.  But I still don't think it's a good docstring.